integration-cli/docker_cli_push_test.go

```diff
@@ -105,10 +105,12 @@ func testPushMultipleTags(c *testing.T) {
 }
 
 func (s *DockerRegistrySuite) TestPushMultipleTags(c *testing.T) {
+	testRequires(c, DockerCLIVersionLessThan(c, "20.10.0"))
```

Comment:
We can add `-a` / `--all-tags` on 20.10 if we want to test it

```
Usage:  docker push [OPTIONS] NAME[:TAG]

Push an image or a repository to a registry

Options:
  -a, --all-tags                Push all tagged images in the repository
```

Reply:
Updated to add the flag when using this cli version instead of skipping.